include/measurement_kit/mlabns/mlabns.hpp

```diff
@@ -19,7 +19,7 @@ MK_DEFINE_ERR(MK_ERR_MLABNS(0), InvalidPolicyError, "")
 MK_DEFINE_ERR(MK_ERR_MLABNS(1), InvalidAddressFamilyError, "")
 MK_DEFINE_ERR(MK_ERR_MLABNS(2), InvalidMetroError, "")
 MK_DEFINE_ERR(MK_ERR_MLABNS(3), InvalidToolNameError, "")
-MK_DEFINE_ERR(MK_ERR_MLABNS(4), UnexpectedHttpStatusCodeError, "")
+//MK_DEFINE_ERR(MK_ERR_MLABNS(4), UnexpectedHttpStatusCodeError, "")
```

Comment:
Is it ok to have this line commented out?

If we are not using it anymore (because `HttpRequestFailedError` is used instead), maybe we can just delete it?